__tests__/app.test.js

```diff
@@ -27,8 +27,41 @@ describe("/api", () => {
   test("GET: 200 responds with an object describing all available endpoints on API", () => {
     return request(app)
       .get("/api")
-      .then(() => {
-        expect(endpoints).toEqual(endpoints);
+      .then((result) => {
+        expect(result.body.endpoints).toEqual(endpoints);
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET: 200 responds with an object containing properties of author, title, article_id, body, topic, created_at, votes, article_img_url", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body[0].article_id).toBe(1)
```

Comment:
I'd recommend using  jest matchers such as **toMatchObject or objectContaining** keep the tests flexible if the test data was to change or extend in properties in the future.